blockchain/blockchain.go

```diff
@@ -340,7 +340,7 @@ func (b *Blockchain) Store(block *core.Block, blockCommitments *core.BlockCommit
 		if err := verifyBlock(txn, block); err != nil {
 			return err
 		}
-		return errors.New("temp error")
```

Comment:
??

Reply:
I added this temporary error in my branch to prevent node from storing block, until I'm sure that all necessary data is passed